src/interactions/map_start/trashBin.interaction.js

```diff
@@ -0,0 +1,16 @@
+import { displayDialogue } from '../../utils';
+
+export const interactionWithTrashBin = (player, k, map) => {
+    player.onCollide('trash_bin_lobby', () => {
```

Comment:
Can you use like you mentioned the closet. It is a bit weird named by me 🙈. The tag is called `cabin_edge_room_1`
You can find it [here] (https://github.com/zero-to-mastery/ZTM-Quest/blob/main/additional-map-info/map_start/objects_map_start_1.png)

Reply:
Changed naming scheme, also fixed nameing error in map_start.json()